architecture-standards/0001-use-architectural-decision-records.md

```diff
@@ -23,22 +23,23 @@ The main logic with ADRs is to describe (architectural) decisions made:
 
 * To provide best practices and solutions we (as *build tool* team) want to promote.
 * To avoid asking the same thing multiple times during code review.
-* To explain *rejected solutions*, for now and future development, in case of re-visit.
+* To explain *rejected solutions*, for now and future development, in case they are proposed again.
 
-ADRs can be written by any team, like code, it has to be reviewed by any other relevant teams.
-The goal is not to *own*, but to *share* with other teams, to improve the build tool conjointly.
+ADRs can be written by any team.
+Like code, they should be reviewed by any other relevant teams.
+The goal is not to *own* a decision, but to *share* it with other teams, and improve the build tool together.
 
 
 ### Format
 
-The format for ADR in this module should follow this template:
+The format for ADR should follow this template:
 
 ```markdown
 # ADR-000X - Title
 
 ## Date
 
-20XY-AB-CD
+20YY-MM-DD
```

Comment:
🤔 The date here is the proposed date.  Should we also track an *accepted* date?

Reply:
No, I don't think that's that important